extensions/arc/src/localizedConstants.ts

```diff
@@ -87,7 +88,7 @@ export const restoring = localize('arc.restoring', "Restoring");
 export const recovering = localize('arc.recovering', "Recovering");
 export const recoveryPending = localize('arc.recoveringPending', "Recovery Pending");
 export const suspect = localize('arc.suspect', "Suspect");
-export const emergecy = localize('arc.emergecy', "Emergecy");
+export const emergency = localize('arc.emergency', "Emergency");
```

Comment:
While I do appreciate you fixing these - in the future please do unrelated fixes in separate PRs.

In general the rules I try to follow are :

1. If I see a mistake in a file I'm already including that's fine as long as it's only one or two two small things
2. Otherwise just open a separate PR so reviewers don't have to spend time trying to figure out what changes are important and actually part of the changes being proposed.

Reply:
Ack.